### ISSUE TITLE
example can't worked

### ISSUE THREAD

**Issue by Vi-jay** [react-native-community/react-native-tab-view#514]
```javascript
renderScene = ({ route }) => {
  if (Math.abs(this.state.index - this.state.routes.indexOf(route)) > 2) {
    return null;
  }

  return <MySceneComponent route={route} />;
};
```
if `renderScene` return null ,will be not render anything again

**Comment by ericketts** [comment]
doesn't get much more optimised than that!

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.
